api/src/main/java/edu/cornell/mannlib/vitro/webapp/migration/auth/AuthMigrator.java

```diff
@@ -72,6 +73,33 @@ public void contextInitialized(ServletContextEvent sce) {
         if (!isMigrationRequired()) {
             return;
         }
+        long currentVersion = getVersion();
```

Comment:
Can you please refactor your code that getVersion() is invoked only once? You are executing the same Sparql query in some cases three times.

Reply:
fixed